src/enforce/runners/any_of/index.js

```diff
@@ -16,7 +16,7 @@ export default function anyOf(value: mixed, tests: Tests, rules: Rules): boolean
 
         if (success === true) {
             successCount++;
```

Comment:
Now `successCount` can be turned into a `boolean`, and the function can simply return it so:
`return isSuccessful;` (better naming may be needed)

instead of the current:
`return successCount > 0;`